src/widget/OktaSignIn.js

```
@@ -80,15 +80,34 @@ var OktaSignIn = (function () {
      * @param success - success callback function
      * @param error - error callback function
      */
+    var router;
     function render(options, success, error) {
-      var router = new LoginRouter(_.extend({}, config, options, {
+      router = new LoginRouter(_.extend({}, config, options, {
         authClient: authClient,
         globalSuccessFn: success,
         globalErrorFn: error
       }));
       router.start();
     }
 
+    function hide() {
+      if (router) {
+        router.hide();
+      }
+    }
+
+    function show() {
+      if (router) {
+        router.show();
```

I don't think we should throw an error if the router doesn't exist.